dockers/docker-database/supervisord.conf.j2

```diff
@@ -33,3 +33,11 @@ stdout_logfile=syslog
 stderr_logfile=syslog
 {%     endfor %}
 {% endif %}
+
+[program:flushdb]
```

Comment:
>flushdb [](start = 9, length = 7)

Can we delay it like 5 min? so it will not impact warm-reboot time.

Reply:
we don't need to wait. This process is not in warmboot sequence. warmboot go ahead once database docker is started.

Reply:
But it still burns CPU.

---
In reply to: [520939272](https://github.com/Azure/sonic-buildimage/pull/5773#discussion_r520939272) [](ancestors = 520939272)

Reply:
add sleep 300